README.md

```diff
@@ -386,12 +386,25 @@ Or with JUnit 5 :
 
 ```
 @ExtendWith(LocalstackDockerExtension.class)
-@LocalstackDockerProperties(randomizePorts = true, services = { "sqs", "kinesis:77077" })
+@LocalstackDockerProperties(services = { "sqs", "kinesis:77077" })
 public class MyDockerCloudAppTest {
    ...
 }
 ```
 
+You can configure the Docker behaviour using the `@LocalstackDockerProperties` annotation with the following parameters:
+
+| property                    | usage                                                                                                                        | type                         | default value |   |
+|-----------------------------|------------------------------------------------------------------------------------------------------------------------------|------------------------------|---------------|---|
+| pullNewImage                | Determines if a new image is pulled from the docker repo before the tests are run.                                           | boolean                      | false         |   |
+| randomizePorts              | Determines if the container should expose the default local stack ports (4567-4583) or if it should expose randomized ports. | boolean                      | false         |   |
+| services                    | Determines which services should be run when the localstack starts.                                                          | String[]                     | All           |   |
+| imageTag                    | Use a specific image tag for docker container                                                                                | String                       | latest        |   |
+| hostNameResolver            | Used for determining the host name of the machine running the docker containers so that the containers can be addressed.     | IHostNameResolver            | localhost     |   |
+| environmentVariableProvider | Used for injecting environment variables into the container.                                                                 | IEnvironmentVariableProvider | Empty Map     |   |
+
+_NB : When specifying the the port in the `services` property, you cannot use `randomizePorts = true`_
+
 The *LocalStack* JUnit test runner is published as an artifact in Maven Central.
 Simply add the following dependency to your `pom.xml` file:
```

Comment:
I'd move the new table below the following paragraph. Configuring the dependency in `pom.xml` seems like higher priority than the `@LocalstackDockerProperties` customizations.